testkit/src/main/scala/geotrellis/contrib/testkit/BetterRasterMatchers.scala

```diff
@@ -48,7 +50,7 @@ trait BetterRasterMatchers { self: Matchers with FunSpec with RasterMatchers =>
   def assertTilesEqual(actual: MultibandTile, expected: MultibandTile): Unit = {
     actual should have (
       cellType (expected.cellType),
-      dimensions (expected.dimensions),
+      // dimensions (expected.dimensions),
```

Comment:
Hm, why is it commented out?

Reply:
Type checking on that was not working out well. The exact reason is to do with how ScalaTest matchers are being constructed, where the thing being checked is in separate param block from the thing doing the checker. Now that dimensions of `CellGrid[N]` are `N` I had hell of a time getting that matcher to work. Basically something I bailed on to come back and re-evaluate how/if those kind of matchers are worth while.